### ISSUE TITLE
Installing and Updating ElvUI is broken (Tukui too)

### ISSUE THREAD

**Issue by BSerpent86** [layday/instawow#87]
When trying to install or update (or even reconcile) ElvUI, there is an error:
```
/home/XXXX/.local/lib/python3.9/site-packages/instawow/models.py:94: SAWarning: Column 'pkg_folder.name' is marked as a member of the primary key for table 'pkg_folder', but has no Python-side or server-side default generator indicated, nor does it indicate 'autoincrement=True' or 'nullable=True', and no explicit value is passed.  Primary key columns typically may not store NULL.
  connection.execute(
! tukui:elvui
  internal error: "(sqlite3.IntegrityError) NOT NULL constraint
    failed: pkg_folder.name [SQL: INSERT INTO pkg_folder DEFAULT
    VALUES] (Background on this error at:
    https://sqlalche.me/e/14/gkpj)"
```
According to the Tukui Forums there has been a change to the addon package that needs updates on updater tools, even their own updater tool is broken because of that change:
https://www.tukui.org/forum/viewtopic.php?f=5&t=17239
I am way too noobish in programming to see if that change is actually the source of the error.
The error happens for at least 2 ElvUI updates now.

Running instawow 1.31.3 installed via pip on Arch stable channel.

**Comment by layday** [comment]
Fixed in 1.32.0 which I've just published on PyPI.

Thanks for the report!

**Comment by BSerpent86** [comment]
thanks too, works again
